extensions/mongodb/src/main/java/com/hazelcast/jet/mongodb/MongoSinkBuilder.java

```diff
@@ -244,7 +245,22 @@ public MongoSinkBuilder<T> writeMode(@Nonnull WriteMode writeMode) {
      */
     @Nonnull
     public MongoSinkBuilder<T> throwOnNonExisting(boolean throwOnNonExisting) {
-        params.setThrowOnNonExisting(throwOnNonExisting);
+        params.setCheckExistenceOnEachConnect(throwOnNonExisting);
+        return this;
+    }
+
+    /**
+     * If {@link ResourceChecks#NEVER}, the database and collection will be automatically created on the first usage.
+     * Otherwise, querying for a database or collection that don't exist will cause an error.
+     * Default value is {@link ResourceChecks#ONCE_PER_JOB}.
```

Comment:
>Default value is {@link ResourceChecks#ONCE_PER_JOB}

nope, default in this builder is null, but it has the same ultimate result.

Reply:
I've made it explicit